client/public/index.html

```diff
@@ -30,7 +30,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>NLP Sandbox PHI Deidentifier</title>
+    <title>NLP Sandbox PHI De-identifier</title>
```

Comment:
It seems that the preferred title from the poll on Discord is `NLP Sandbox PHI Deidentifier`.

I agree with using "de-identification", "de-identify", etc.

Reply:
Got it. Changed it back now